doc/rbd/rbd-snapshot.rst

```diff
@@ -4,22 +4,24 @@
 
 .. index:: Ceph Block Device; snapshots
 
-A snapshot is a read-only copy of the state of an image at a particular point in
-time. One of the advanced features of Ceph block devices is that you can create
-snapshots of the images to retain a history of an image's state. Ceph also
-supports snapshot layering, which allows you to clone images (e.g., a VM image)
-quickly and easily. Ceph supports block device snapshots using the ``rbd`` 
-command and many higher level interfaces, including `QEMU`_, `libvirt`_, 
-`OpenStack`_ and `CloudStack`_.
+A snapshot is a read-only logical copy of an image at a particular point in
+time, aka a checkpoint. One of the advanced features of Ceph block devices is
```

Comment:
I kind of want to change "aka" to "equivalent to" or "which can be thought of as". I know that both of those are much longer, but we're not paying by the pixel, so unless there's some well-camouflaged belletristic reason to keep "aka" here, let's change it.

Reply:
Tweaked slightly